apps/hub/src/domains/game/components/game-backend/game-backend-event-summary-logs.tsx

```diff
@@ -0,0 +1,44 @@
+import { LogsView, Tabs, TabsList, TabsTrigger } from "@rivet-gg/components";
+import { useState } from "react";
+import type { BackendEvent } from "../../queries";
+
+type LogType = "std_out" | "std_err";
+
+interface GameBackendEventSummaryLogsProps
+  extends Pick<
+    BackendEvent,
+    "logs" | "logTimestamps" | "exceptions" | "exceptionsTimestamps"
+  > {}
+
+export function GameBackendEventSummaryLogs({
+  logs,
+  logTimestamps,
+  exceptions,
+  exceptionsTimestamps,
+}: GameBackendEventSummaryLogsProps) {
+  const [logType, setLogType] = useState<LogType>("std_out");
+
+  const lines = logType === "std_out" ? logs : exceptions;
+  const timestamps =
+    logType === "std_out" ? logTimestamps : exceptionsTimestamps;
+
+  return (
+    <>
+      <Tabs
+        value={logType}
+        onValueChange={(value) => setLogType(value as LogType)}
+      >
+        <TabsList>
+          <TabsTrigger value="std_out">stdout</TabsTrigger>
+          <TabsTrigger value="std_err">stderr</TabsTrigger>
+        </TabsList>
```

Comment:
This isn't quite right. There's not a stderr for workers.

Excepts are always at the end of the worker.

A couple ideas:


1. Show a red box above the logs with the exception info.

2. Display it similar to GitHub Actions with a line in red.


<img width="1445" alt="image" src="https://github.com/rivet-gg/hub/assets/3322446/439157b7-717b-4a46-9a87-ca8efe015883">

Open to alternative ideas.